src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java

```diff
@@ -60,6 +60,7 @@ public void testUpdate() {
 		productCatalogueService.update(productCatalogue);
 		ProductCatalogue updatedProductCatalogue = productCatalogueService.getProductCatalogue(1l);
 		assertEquals("Updated Product Name", updatedProductCatalogue.getProductName());
+		assertEquals("", updatedProductCatalogue.getPhotoURL());
```

Comment:
Can we update the URL and test here

Reply:
URL can not be updated, as URL points to the end-point /multimedia/media/{entity-id} to download the image file where entity-id is equaled to the unique id of the product catalogue.

Reply:
Yes and that why I dont think the assert makes sense. I was suggesting 
productCatalogue.setPhotoURL("some url");

and the here you assert that the URL is saved

Reply:
Done